### PR TITLE
[opentelemetry-demo] Update demo chart to 0.6.0-beta

### PR DESCRIPTION
**Pull request by joshleecreates**
Fixes https://github.com/open-telemetry/opentelemetry-helm-charts/issues/426

There are errors coming from recommendation service for me using the current version as well as this version. Need to investigate.

### CODE REVIEW

**Comment by joshleecreates** [comment]
Ok, so I have confirmed that the errors I'm seeing on the recommendation service is an issue with the current version and not new. I will make a separate issue for that and I don't think it should block this PR, but I defer to any reviewer who disagrees.

**Comment by TylerHelmuth** [comment]
@joshleecreates no need to block the bump if it's an issue with the demo itself and not it's usage in k8s. The issue appears in docker compose as well?

**Comment by joshleecreates** [comment]
> @joshleecreates no need to block the bump if it's an issue with the demo itself and not it's usage in k8s. The issue appears in docker compose as well?

It does not occur in docker compose. It does occur with the current main commit of the helm chart.

**Comment by TylerHelmuth** [comment]
Thanks for creating the issue. I think we can move forward with the bump then and prioritize fixing the issue in a separate PR.

File: charts/opentelemetry-demo/Chart.yaml
Comment (TylerHelmuth) [inline charts/opentelemetry-demo/Chart.yaml:4]:
```suggestion
version: 0.7.0
```
Normally we do appVersion bumps as a minor bump.

Comment (TylerHelmuth) [reply charts/opentelemetry-demo/Chart.yaml:4]: You'll have to regenerate the examples.

**Review by TylerHelmuth — APPROVED** [review]
Thanks for doing this.

**Pull request merged**
